nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/GetAzureEventHub.java

```diff
@@ -121,6 +122,20 @@
             .expressionLanguageSupported(false)
             .required(false)
             .build();
+    static final PropertyDescriptor RECEIVER_FETCH_SIZE = new PropertyDescriptor.Builder()
+            .name("Partition Recivier Fetch Size")
+            .description("The number of events that a receiver should fetch from an EventHubs partition before returning. Default(100)")
+            .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
+            .expressionLanguageSupported(false)
+            .required(false)
```

Comment:
Consider making it required with default value set to 100. This will make the code a bit simpler especially when it gets to line:288 where you can eliminate the entire IF statement and simply assign.

The same goes for the next property (RECEIVER_FETCH_TIMEOUT) and line:224